pkg/rust/cargo/testdata/cargo_invalid.lock

```diff
@@ -0,0 +1,34 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
+[[package]]
+name = "libc"
+version = "0.2.54"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+
+[[package]]
+name = "normal"
+version =
+dependencies = [
+ "libc 0.2.54 (registry+https://github.com/rust-lang/crates.io-index)",
+]
+
+[[package]]
+name = "typemap"
+version = "0.3.3"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+dependencies = [
+ "unsafe-any 0.4.2 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
`unsafe-any` is not on the package list. Is it possible?

Reply:
OK, this is an invalid case, but how about `pkg/rust/cargo/testdata/cargo_normal.lock` and `pkg/rust/cargo/testdata/cargo_mixed.lock`? `unsafe-any` is in dependencies, but I didn't find it in package list. I want to use a valid cargo.lock for valid cases.